release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -404,6 +404,53 @@ In {product-title} {product-version} and later, you can edit the baseboard manag
+
+* kube-state-metrics to 2.12.0
+* Metrics Server to 0.7.1
+* node-exporter to 1.8.0
+* Prometheus to 2.52.0
+* Prometheus Operator to 0.73.2
+* Thanos to 0.35.0
+
+[id="ocp-4-16-monitoring-changes-to-alerting-rules"]
+==== Changes to alerting rules
+
+[NOTE]
+====
+Red{nbsp}Hat does not guarantee backward compatibility for recording rules or alerting rules.
+====
+
+* Added the `ClusterMonitoringOperatorDeprecatedConfig` alert to monitor when the Cluster Monitoring Operator configuration uses a deprecated field.
+* Added the `PrometheusOperatorStatusUpdateErrors` alert to monitor when the Prometheus Operator fails to update object status.
+
+[id="ocp-4-16-monitoring-metrics-server-to-access-metrics-api-ga"]
+==== Metrics Server component to access the Metrics API general availability (GA)
+
+Metrics Server component is now generally available and automatically installed instead of the deprecated Prometheus Adapter. Metrics Server collects resource metrics and exposes them in the `metrics.k8s.io` Metrics API service for use by other tools and APIs, which frees the core platform Prometheus stack from handling this functionality. For more information, see xref:../observability/monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc#metricsserverconfig[MetricsServerConfig] in the config map API reference for the Cluster Monitoring Operator.
+
+[id="ocp-4-16-monitoring-new-role-to-allow-read-only-access-to-alertmanager-api"]
+==== New monitoring role to allow read-only access to the Alertmanager API
+
+This release introduces a new `monitoring-alertmanager-view` role to allow read-only access to the Alertmanager API in the `openshift-monitoring` project.
+
+[id="ocp-4-16-monitoring-vpa-metrics-available-in-kube-state-metrics-agent"]
+==== VPA metrics are available in the kube-state-metrics agent
+
+Vertical Pod Autoscaler (VPA) metrics are now available through the `kube-state-metrics` agent. VPA metrics follow a similar exposition format like they did before being deprecated and removed from native support upstream.
+
+[id="ocp-4-16-monitoring-change-in-proxy-service-for-monitoring-components"]
+==== Change in proxy service for monitoring components
+With this release, the proxy service in front of Prometheus, Alertmanager, and Thanos Ruler has been updated from OAuth to `kube-rbac-proxy`. This change might affect service accounts and users accessing these API endpoints without the appropriate roles and cluster roles.
+
+[id="ocp-4-16-monitoring-change-in-how-prometheus-handles-duplicate-samples"]
+==== Change in how Prometheus handles duplicate samples
+With this release, when Prometheus scrapes a target, duplicate samples are no longer silently ignored, even if they have the same value. The first sample is accepted and the `prometheus_target_scrapes_sample_duplicate_timestamp_total` counter is incremented, which might trigger the `PrometheusDuplicateTimestamps` alert.
```

Comment:
I don't think we need to change, we can keep as it was.
reason:
4.15 and 4.16, PrometheusDuplicateTimestamps alert is the same definition. I think the purpose to mention the change for PrometheusDuplicateTimestamps alert is it will add runbook_url after https://github.com/openshift/cluster-monitoring-operator/pull/2365 back ported to 4.16. @machine424 what's your suggestion
```
$ oc get clusterversion
NAME      VERSION                              AVAILABLE   PROGRESSING   SINCE   STATUS
version   4.15.0-0.nightly-2024-06-19-045027   True        False         10h     Cluster version is 4.15.0-0.nightly-2024-06-19-045027

$ oc -n openshift-monitoring get prometheusrules prometheus-k8s-prometheus-rules -ojsonpath='{.spec.groups[].rules[?(@.alert=="PrometheusDuplicateTimestamps")]}' | jq
{
  "alert": "PrometheusDuplicateTimestamps",
  "annotations": {
    "description": "Prometheus {{$labels.namespace}}/{{$labels.pod}} is dropping {{ printf \"%.4g\" $value  }} samples/s with different values but duplicated timestamp.",
    "summary": "Prometheus is dropping samples with duplicate timestamps."
  },
  "expr": "rate(prometheus_target_scrapes_sample_duplicate_timestamp_total{job=~\"prometheus-k8s|prometheus-user-workload\"}[5m]) > 0\n",
  "for": "1h",
  "labels": {
    "severity": "warning"
  }
}
```